JustAnotherVoiceChat.Server.Wrapper/src/Elements/Client/VoiceClient.cs

```diff
@@ -66,6 +68,16 @@ public bool SetNickname(string nickname)
             return Server.NativeWrapper.SetClientNickname(this, nickname);
         }
 
+        private T RunWhenConnected<T>(Func<T> callback)
+        {
+            if (!Connected)
+            {
+                throw new ClientNotConnectedException(Handle);
```

Comment:
I think, throwing an exception here is maybe the wrong decision, because most of the times, where we use this method, we return a `bool` that indicates if the method has been executed successfully. Maybe `default(T)` as return value would be better.